app/retail/templates/shared/top_nav.html

```diff
@@ -20,7 +20,7 @@
 <!--- Top Row Notification -->
 <div class="row bottom_notification font-caption" style="text-align: center;">
   <span style="text-align: center; margin: auto;">
-    {% blocktrans %}<strong>The Beyond Blocks Hackathon was just announced!</strong> - A three week virtual hackathon where global developers + entrepeneurs will collaborate to push blockchain applications to new frontiers of business + technology + social change <a href="https://gitcoin.co/hackathon/beyondblocks">Sign up today</a>.{% endblocktrans %}
+    {% blocktrans %}<strong>The Beyond Blockchain Hackathon was just announced!</strong> - A three week virtual hackathon where global developers + entrepeneurs will collaborate to push blockchain applications to new frontiers of business + technology + social change <a href="https://gitcoin.co/hackathon/beyondblocks">Sign up today</a>.{% endblocktrans %}
```

Comment:
should we edit the `top_nav_salt` and update the message to say "The Beyond Blockchain Hackathon *has just started*" ?  that way ppl will see that it started today...